README.md

```diff
@@ -9,6 +9,8 @@ in JupyterLab and the Jupyter Notebook. More specifically, Jupyter AI offers:
 * A native chat UI in JupyterLab that enables you to work with generative AI as a conversational assistant.
 * Support for a wide range of generative model providers, including AI21, Anthropic, AWS, Cohere,
   Hugging Face, and OpenAI.
+* Local model support through GPT4All, enabling use of generative AI models on consumer grade machines
+  with ease and without compromising privacy.
```

Comment:
```suggestion
* Local model support through GPT4All, enabling use of generative AI models on consumer grade machines
  with ease and privacy.
```

Let's not imply that using a 3P LLM compromises privacy

Reply:
Updated.